### PR TITLE
Jose Lopez

### PR DESCRIPTION
**Pull request by JoseLopez235**

### CODE REVIEW

**Comment by JoseLopez235** [comment]
Sorry I turned it a min late when i made a pull request the first time I made a mistake and was trying to fix it

File: lib/character.rb [hunk 1]
```diff
@@ -0,0 +1,20 @@
+class Character
+  attr_reader :name, :actor, :salary
+  def initialize(character_hash)
+    @name = character_hash[:name]
+    @actor = character_hash[:actor]
+    @salary = character_hash[:salary]
+  end
+
+  def name_all_caps?
+    name_all_caps = false
```
Comment (mikedao) [inline lib/character.rb:10]: I think a simpler way is `@name.upcase == @name`

Comment (mikedao) [reply lib/character.rb:10]: good refactor though

File: lib/network.rb [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+      show.characters.each do |character|
+        if shows_hash[show.name].nil?
+          shows_hash[show.name] = [character.actor]
+        else
+          shows_hash[show.name] << character.actor
+        end
+      end
+    end
+    shows_hash
+  end
```
Comment (mikedao) [inline lib/network.rb:37]: The key is supposed to be the show object, not the show name.

**Pull request closed without merging**